scripts/check-go-version.sh

```diff
@@ -79,7 +84,26 @@ function check() {
 check go.mod "^go\s\+$version_regex" "$go_previous_version"
 check internal/tools/go.mod "^go\s\+$version_regex" "$go_latest_version"
 
-check docker/debug/Dockerfile "^.*golang:$version_regex" "$go_latest_version"
+# Patch Version Handling in Dockerfile
+check_dockerfile() {
```

Comment:
The original script was already detecting where changes were needed, it just applied them incorrectly. Why is this function needed?

Reply:
yes, you are right, I thought to implement it specifically for docker as a function but yeah it is not needed. (also the incorrect is the sed command part? {too noob to detect the err})